nearcache/nearcache_it_test.go

```diff
@@ -73,6 +77,80 @@ func TestSmokeNearCachePopulation(t *testing.T) {
 	})
 }
 
+func TestGetAllChecksNearCacheFirst(t *testing.T) {
+	// port of: com.hazelcast.client.map.impl.nearcache.ClientMapNearCacheTest#testGetAllChecksNearCacheFirst
+	tcx := newNearCacheMapTestContext(t, nearcache.InMemoryFormatObject, false)
+	tcx.Tester(func(tcx it.MapTestContext) {
+		t := tcx.T
+		m := tcx.M
+		ctx := context.Background()
+		var keys []interface{}
+		var target []types.Entry
+		const size = 1003
+		for i := int64(0); i < size; i++ {
+			if _, err := m.Put(ctx, i, i); err != nil {
+				t.Fatal(err)
+			}
+			keys = append(keys, i)
+			target = append(target, types.Entry{Key: i, Value: i})
+		}
+		// populate Near Cache
+		for i := int64(0); i < size; i++ {
+			v, err := m.Get(ctx, i)
+			if err != nil {
+				t.Fatal(err)
+			}
+			require.Equal(t, i, v)
+		}
+		// GetAll() generates the Near Cache hits
+		vs, err := m.GetAll(ctx, keys...)
+		if err != nil {
+			t.Fatal(err)
+		}
+		sort.Slice(vs, func(i, j int) bool {
```

Comment:
I think we can use `require.ElementsMatch` instead of sorting

Reply:
Makes sense: https://github.com/hazelcast/hazelcast-go-client/pull/826/commits/288838714331c22b574ef533bb32054d9c682ddd